### PR TITLE
Use color picker library to fix problems with Firefox

### LINKED ISSUE #6487: Changing segment/tree color is broken in Firefox
When using the context menu on a segment or tree (in its respective tab), the input picker is a new window in firefox which makes the context menu close. Therefore, changing the color in the picker doesn't have any effect (since the responsible component was unmounted).

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://fixcolorpicker.webknossos.xyz/

### Steps to test:
- create a new annotation
- create a tree and change it's color via right click in the tree tab
- create a segment and change it's color via right click in the segments tab
- ensure that the color editing "feels good" (the popover should close when it loses focus etc)

### Issues:
- fixes #6487

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [X] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
it's now deployed at https://fixcolorpicker.webknossos.xyz/

File: frontend/javascripts/components/color_picker.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Popover } from "antd";
+import * as Utils from "libs/utils";
+import { HexColorInput, HexColorPicker } from "react-colorful";
+import useThrottledCallback from "beautiful-react-hooks/useThrottledCallback";
```
Comment (hotzenklotz) [inline frontend/javascripts/components/color_picker.tsx:5]: Cool library 🤩

**Review by hotzenklotz — APPROVED** [review]
Nice. Works like charm. Test with FF, Safari and Chrome on Mac.

File: frontend/javascripts/components/color_picker.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+    const colorRgb = Utils.hexToRgb(colorStr);
+    const newColor = Utils.map3((component) => component / 255, colorRgb);
+    onSetColor(newColor);
+  };
+  const content = isDisabled ? null : (
+    <ThrottledColorPicker color={color} onChange={onChangeColor} />
+  );
+  return (
+    <Popover content={content} trigger="click" overlayStyle={{ zIndex: 10000 }}>
```
[the remark this answers is not in the document]
Comment (philippotto) [reply frontend/javascripts/components/color_picker.tsx:67]: :laughing: that's capturing how I felt while adding that really well

**Pull request merged**